### ISSUE TITLE
Could Let's Encrypt extension automatically get new certificate when I add new custom domain

### ISSUE THREAD

**Issue by huulocnguyen** [sjkp/letsencrypt-siteextension#90]
Hi @sjkp
If I add new custom domain to web app service, so do Let's Encrypt extension support to get certificate for new custom domain automatically ? Or is there any way to support to do this thing automatically? 

Thank you so much!

**Comment by sjkp** [comment]
hi @huulocnguyen 

the way the extension works is that a web job (running in your site) regularly checks if the certificates needs updating and if it does, it will request the certificate and install it for you. So it is automated. 

However there are several situations in which the web jobs can get deleted when you deploy your own website (e..g if you select delete files at destination), and under those circumstances you will have to either reinstall the site extension or just renew the certificate manually (which takes 30sec after the initial setup).

**Comment by huulocnguyen** [comment]
Hi @sjkp 
Thank for your quickly reply!
I have configured successfully let's encrypt extension and it work well for domain which I registered in site extention (e.g wedding.mydomain.com). But now when my staff add one more new custom domain in azure(e.g pizza.mydomain.com). so can let's encrypt extension recognise new custom domain, and automatically  get new certificate for pizza.mydomain.com. and I don't need to go to site extension again to register a new one manually.
